---
aliases:
  - Molecular typing methods
tags:
  - Topics/microorganisms
---
determines the [[Nucleotide]] composition of the full or partial genome for the classification of [[Microorganism|microorganisms]]

# PCR-based methods 
amplify (i.e.  making multiple copies) [[Deoxyribonucleic Acid|DNA]] or RNA sequences from an organism's genome
## Specific Amplification
amplify specific segments of the genome (genes, parts of genes, or even noncoding regions)
### Techniques for Amplification
#### Polymerase Chain Reaction (PCR)
#### Quantitative PCR (qPCR)
#### Reverse Transcription PCR (RT-PCR)
### Analysis Methods
#### Direct Methods
detects the presence or absence of expected gene fragment(s) 
#### Indirect Methods
analyzes fragments post-digestion with restriction enzymes
##### Restriction Fragment Length [[Polymorphism]] (RFLP)
uses restriction enzymes, which cut the DNA at specific sequences, to generate fragments of different length that can be separated on an electrophoresis gel → resulting pattern (RFLP) is unique to the microorganism
genes and restriction enzymes choice affects the technique's ability to distinguish between different organisms or strains
###### Ribotyping 
Targets the 16S rRNA gene, a component of the ribosome found universally in bacteria to distinguish between species or strains

*De Cesare, A., Bruce, J. L., Dambaugh, T. R., Guerzoni, M. E., and Wiedmann, M. (2001a). Automated ribotyping using different enzymes to improve discrimination of Listeria monocytogenes isolates, with a particular focus on serotype 4b strains. J. Clin. Microbiol. 39, 3002–3005. doi: 10.1128/jcm.39.8. 3002-3005.2001* 
*De Cesare, A., Manfreda, G., Dambaugh, T. R., Guerzoni, M. E., and Franchini, A. (2001b). Automated ribotyping and random amplified polymorphic DNA analysis for molecular typing of Salmonella enteritidis and Salmonella typhimurium strains isolated in Italy. J. Appl. Microbiol. 91, 780–785. doi: 10.1046/j.1365-2672.2001.01441.x*
###### Fla-typing ([[Campylobacter]])
argets the flagellin (fla) gene, which codes for the structure of flagella to differentiate strains within _Campylobacter_

*Djordjevic, S. P., Unicomb, L. E., Adamson, P. J., Mickan, L., and Rios, R. (2007). Clonal complexes of campylobacter jejuni identified by multilocus sequence typing are reliably predicted by restriction fragment length polymorphism analyses of the flaA gene. J. Clin. Microbiol. 45, 102–108. doi: 10.1128/jcm. 01012-06*
###### InvA/FliC-typing ([[Salmonella]])
Targets the the _invA_ gene (invasion of host cells) or the FliC gene (flagella structure) to differentiate strains within *Salmonella*

##### Amplified Fragment Length [[Polymorphism]] (AFLP)
begins with genome restriction followed by amplification of the produced fragments
*Wang, B., Wang, C., McKean, J. D., Logue, C. M., Gebreyes, W. A., Tivendale, K. A., et al. (2011). Salmonella enterica in swine production: assessing the association between amplified fragment length polymorphism and epidemiological units of concern. Appl. Environ. Microbiol. 77, 8080–8087. doi: 10.1128/AEM.00064-11*

##### Molecular Serotyping
for certain pathogens like _[[Listeria]] monocytogenes_, PCR has been adapted to replace [[Phenotypic typing methods#Serotyping / Serovar Determination|traditional serotyping]] methods with molecular-based approaches
*Doumith, M., Buchrieser, C., Glaser, P., Jacquet, C., and Martin, P. (2004). Differentiation of the major listeria monocytogenes serovars by multiplex PCR. J. Clin. Microbiol. 42, 3819–3822. doi: 10.1128/jcm.42.8.3819-3822.2004*

##### Detection in Foodborne [[Viruses]]
Real-time RT-PCR (RT-qPCR) and digital PCR (dPCR) are the gold standards, allowing for the detection and basic typing of viruses such as [[Norovirus]] Genogroup I or II by using group-specific probes

## Non-specific Amplification 
generates randomly amplified DNA fragments from the entire genome without targeting specific gene sequences using degenerate primers and/or sub-string PCR conditions
### Method
- PCR under these conditions aims to produce a diverse but reproducible set of DNA fragments within a laboratory setting
- Size polymorphisms among the amplified fragments are revealed through conventional agarose gel electrophoresis
  → Random Amplified Polymorphic DNA (RAPD) profile
- RAPD profiles are used to differentiate between strains of a microorganism

### Limitations
- Reproducibility Issues: Challenges in achieving consistent results across different settings
- Comparative Analysis: Limited utility in comparing large microbial populations over time due to variability in amplification and fragment generation

## Multi-Locus Variable Number Tandem Repeat Analysis (MLVA)
Genetic fingerprinting technique that examines variations in the number of tandem repeat sequences (Variable Number Tandem Repeats (VNTRs)) found within the genomes of bacteria and parasites
*Ferrari, R. G., Panzenhagen, P. H. N., and Conte-Junior, C. A. (2017). Phenotypic and Genotypic eligible methods for Salmonella typhimurium source tracking. Front. Microbiol. 8:2587. doi: 10.3389/fmicb.2017.02587*

### Variable Number Tandem Repeats (VNTRs)
DNA sequences in the genome that are repeated in tandem a certain number of times, varying in the number of repeats among organisms or strains. These sequences are classified as minisatellites or microsatellites, depending on their length.
- Sequence Size Variability: The total length of a VNTR region varies in multiples of the repeated unit
- Multiple VNTR Zones: A single genome may contain several VNTR loci, each with unique sequences

### Analysis
The diversity of the VNTR loci and for each VNTR zone the size of the amplified fragments are assessed through Fragment Size Analysis
- PCR and gel electrophoresis
- capillary sequencing 
- directly by analyzing the sequence 

### Standardization
determining the number of VNTR loci retained in the analysis and considering the number of repeats found at each locus

# Methods based on the Analysis of RFLP
Analysis of RFLP (Restriction Fragments Length Polymorphism) of the entire genome by [[Pulsed-Field Gel electrophoresis]] 

### Method
- **Genome Fragmentation**: Uses restriction enzymes to cut the bacterial genome at rare cleavage sites, resulting in a manageable number of large fragments
- **[[Pulsed-Field Gel electrophoresis|PFGE]]**: Due to their size, these fragments require separation by pulsed-field gel electrophoresis

### Enzyme Selection
- Selecting an appropriate enzyme or combination of enzymes for each bacterium 
	- e.g., AscI/ApaI enzymes for _[[Listeria]] monocytogenes_
- Goal: generate around 15 fragments of variable sizes for different strains
  *Graves, L. M., and Swaminathan, B. (2001). PulseNet standardized protocol for subtyping Listeria monocytogenes by macrorestriction and pulsed-field gel electrophoresis. Int. J. Food Microbiol. 65, 55–62. doi: 10.1016/s0168-1605(00) 00501-8*
- The discriminatory power may be increased by using multiple restriction enzymes in parallel

### Plasmid Analysis 
The same principle can be applied to bacterial [[plasmids]], with enzyme selection tailored to produce fragments suitable for normal agarose gel electrophoresis, yielding a distinctive plasmid profile for each bacterium

### Standardization
- **Comparison**: For accurate strain comparison and source attribution, protocols must be standardized, specifying the choice of enzymes and electrophoresis conditions

### Advantages and Limitations
- [p] PFGE is robust and reproducible across different labs
- [c] PFGE may fall short in distinguishing between very closely related strains, such as those from the same species and individual with minimal genetic evolution

# Methods Based on the Analysis of Genomic Sequences 
Sequencing: determining the exact order of the nucleotides (the building blocks of DNA and RNA) in a specific region of DNA or RNA
## Multi-Locus Sequence Typing  (MLST)
analyzes the [[Allele|allelic]] diversity at selected loci across the bacterial genome
### Methods
- For each selected locus, DNA is sequenced, and specific alleles are identified based on sequence variations. Each unique sequence variant at a locus is assigned a distinct allele number.
- A combination of allele numbers across the chosen loci defines the Sequence Type (ST) of a strain (genetic fingerprint)
- Clonal Complexes (CC) are groups of STs that are closely related, typically sharing alleles at a majority of the loci, indicating evolutionary relationships
### Standardization
- standardized set of loci for each organism or species under study for consistent identification and comparison of STs
- a comprehensive ST database
### [[Viruses|Viral]] Typing
- While less common, MLST can be adapted for viral typing
- Focuses on regions of the viral genome that offer sufficient variability for strain differentiation 
- Sequencing conserved genomic regions that are stable across different viral strains, which flank the more variable sequences, to accurately identify and distinguish between viral strains and subtypes

# CRISPR-Cas System 
sequences the CRISPR regions to differentiate bacterial strains and infer their [[Phylogeny]]
- **CRISPR**: Clustered Regularly Interspaced Short Palindromic Repeats
	- natural adaptive immune system found in bacteria that provides resistance against foreign genetic elements such as plasmids and phages 
	- captures snippets of DNA from invading [[Viruses]] and incorporates them into the CRISPR loci of the bacterial genome as spacers between repeat sequences
	- CRISPR arrays serve as a genetic memory of past infections, enabling bacteria to recognize and neutralize the invaders if they attack again
	- CRISPR regions in the genome are variable, with new spacer acquisitions reflecting exposure to new viral threats, and the loss of obsolete spacers mirroring an evolutionary adaptation 
	  → bacterial strain's evolutionary journey
- **Cas**: CRISPR-associated proteins

# Comparative Genomic Fingerprinting (CGF) / Haplotyping
identifies the presence or absence of key genes within bacterial genomes, generating a distinct genetic fingerprint for each strain
- effective for organisms with high genomic [[Genomic plasticity|plasticity]] because it focuses on a broader set of genes, where traditional typing methods might not provide sufficient resolution to distinguish closely related strains
- 40 genes for [[Campylobacter]], > 80 genes for [[Salmonella]]Typhi
- offers results consistent + with [[#Multi-Locus Sequence Typing (MLST)|MLST]], but with higher discrimination among strains

*Ravel, A., Hurst, M., Petrica, N., David, J., Mutschall, S. K., Pintar, K., et al. (2017). Source attribution of human campylobacteriosis at the point of exposure by combining comparative exposure assessment and subtype comparison based on comparative genomic fingerprinting. PLoS One 12:e0183790. doi: 10.1371/ journal.pone.0183790*

# [[Whole Genome Sequencing]]

# Sources
- Overview and sources mainly taken from: [@mughini-gras.etal2019]