---
aliases: 
tags:
  - Topics/microorganisms
---
classify and identify microorganisms based on observable physical and biochemical characteristics 
- characteristics result from the expression of an organism's genes as well as the influence of the organism's environment
In comparison to [[Genotypic typing methods]] that rely on [[Deoxyribonucleic Acid|DNA]] or RNA analysis, phenotypic typing focuses on how the genetic information of microorganisms manifests in observable traits
# Biotyping
categorizes bacteria within established taxonomic groups (genus, species, and subspecies) into specific biotypes based on their biochemical characteristics, including
- **Metabolic Pathways**: How bacteria metabolize substances to infer energy production and substrate preferences
  Example: Sugar utilization tests determine how bacteria metabolize certain sugars for energy production
- **Enzyme Activities**: Detection of specific enzymes (e.g.lysine decarboxylase) to distinguish between bacteria
- **Biological, Physiological or other biochemical properties**: Traits determined through methods like Gram staining, which identifies cell wall composition, and motility tests, assessing bacterial movement capabilities
→ When at least one or several characteristics are specific for a group within a species/subspecies, a specific biotype can be determined

# Serotyping / Serovar Determination
Serotyping classifies bacteria by cell wall antigens, using agglutination reactions with hyperimmune sera.

## Antigens
**O-Antigens (Somatic Antigens)**
- Polysaccharide chains that are part of the lipopolysaccharide (LPS) layer located in the outer membrane of Gram-negative bacteria
- specific structure of O antigens varies significantly between strains, making them useful for distinguishing among serotypes

**H-Antigens (Flagellar Antigens)**
- associated with the bacterial flagella which are composed of proteins and are responsible for bacterial motility
- variation in flagellar protein composition allows for differentiation among serotypes

**K-Antigens (Capsular Antigens) - occassionally**
- found in the polysaccharide capsule that surrounds some bacteria
- capsule helps protect the bacteria from phagocytosis and other immune responses
- significant for serotyping because they can vary greatly between strains, contributing to differences in virulence and immune evasion

The serotype of a bacterium is often expressed as an antigenic formula, showcasing the specific antigens identified, such as O and H factors for [[Escherichia coli|E. coli]] O157:H7.

## Schemes
Provide a step-by-step procedure that through successive eliminations lead to the single antigenic formula of the serotype
- Example: [White-Kauffmann-Le Minor](https://www.pasteur.fr/sites/default/files/veng_0.pdf) serotyping scheme for [[Salmonella]] which remains reference standards, despite more advanced molecular methods
- many pathogenic bacteria have serotyping schemes more or less used by the scientific community according to their recognized discriminatory characteristics

## Limitation for viral agents
 - serotyping is not applicable to foodborne viruses like [[Norovirus|norovirus]], [[Hepatitis A|hepatitis A]] virus, and hepatitis E virus, primarily because they cannot be cultured in vitro
 - restricts the use of serum neutralization for typing purposes, and currently, no viral antigen detection tests are available for these viruses
 
# Antibiotyping / Antibiotic Resistance Phenotyping
categorize bacteria into resistance profiles (antibiotypes) based on their response to a standardized list of antibiotics, defined by international guidelines (e.g., EUCAST).

## Method
- **Antibiotic List**: a fixed set of antibiotics is used for testing against bacteria
- **Resistance Thresholds**: For each antibiotic on the list a specific concentration thresholds is defined
- **Outcome**: A bacterium is considered resistant if it grows beyond these threshold concentrations

→ the pattern of resistance that a bacterial strain exhibits against this list of antibiotics becomes a defining feature of that strain

## Sources
- *Threlfall, E. J., Fisher, I. S., Ward, L. R., Tschape, H., and Gerner-Smidt, P. (1999). Harmonization of antibiotic susceptibility testing for Salmonella: results of a study by 18 national reference laboratories within the european union-funded enter-net group. Microb. Drug Resist. 5, 195–200. doi: 10.1089/mdr.1999.5.195*
- *David, J. M., Guillemot, D., Bemrah, N., Thébault, A., Brisabois, A., Chemaly, M., et al. (2013a). The Bayesian microbial subtyping attribution model: robustness to prior information and a proposition. Risk Anal. 33, 397–408. doi: 10.1111/j. 1539-6924.2012.01877.x*

# Phage typing
categorize bacteria based on the sensitivity or resistance of a bacterium to [[Bacteriophage|bacteriophages]]

## Method
- **Goal**: identifying whether a bacterial cell possesses receptors that each bacteriophages tested attaches to
- **Lysis Plaque Formation**: The interaction between bacteriophages and bacteria is made visible by the formation of lysis plaques on a bacterial culture. These clear zones indicate areas where bacteria have been lysed (destroyed) by bacteriophages, serving as a direct marker of phage sensitivity.
- relies on a standard and stable panel of phages that determines the phage type and was developed for a limited number of bacterial species (Audurier et al., 1979) or subtypes (Anderson et al., 1977; Glosnicka and Dera-Tomaszewska, 1999)

## Sources
- *Audurier, A., Chatelain, R., Chalons, F., and Piechaud, M. (1979). Bacteriophage typing of 823 Listeria monocytogenes strains isolated in france from 1958 to 1978 (author’s transl). Ann. Microbiol. 130b, 179–189.*
- *Anderson, E. S., Ward, L. R., Saxe, M. J., and de Sa, J. D. (1977). Bacteriophagetyping designations of Salmonella typhimurium. J. Hyg. 78, 297–300. doi: 10.1017/s0022172400056187*
- *Glosnicka, R., and Dera-Tomaszewska, B. (1999). Comparison of two Salmonella enteritidis phage typing schemes. Eur. J. Epidemiol. 15, 395–401.*

# Proteomics
identification, quantification, and analysis of the entire protein content (the proteome) of a cell, tissue, or organism
## Polypeptide Spectra Analysis
identification and analysis of *polypeptide spectra* expressed by a microorganism
provides cost-effective & high-quality data → replacing classical phenotypical methods
### Method
Matrix-Assisted Laser Desorption Time-Of-Flight (MALDI-TOF) Mass Spectrometry
- generates detailed polypeptide spectra for microorganisms, comprising several hundred polypeptides expressed by the microorganism
- spectra serve as unique fingerprints, enabling identification at the genus/species level and, under certain conditions, allowing for strain differentiation and [[#Antibiotyping / Antibiotic Resistance Phenotyping|antibiotyping]]

**Spectra Stability Verification**: the consistency of these spectra across different culture conditions needs to be ensured for reliable identification and comparison

## Sources
- *Shell, W. S., Sayed, M. L., Allah, F. M. G., Gamal, F. E. M., Khedr, A. A., Samy, A. A., et al. (2017). Matrix-assisted laser desorption-ionization-time-of-flight mass spectrometry as a reliable proteomic method for characterization of Escherichia coli and Salmonella isolates. Vet. World 10, 1083–1093. doi: 10.14202/vetworld. 2017.1083-1093*

# Sources
- Overview and sources mainly taken from: [@mughini-gras.etal2019]