---
aliases:
  - WGS
tags:
---
analyzes the complete genomic sequences for comprehensive [[Genotypic typing methods|typing of a microorganism]]

# Method
- requires sophisticated analysis tools to manage the extensive data, with a focus on standardizing bioinformatics pipelines for consistent and accurate comparisons
- allows the identification of the the number and position of [[Single Nucleotide Polymorphism|single nucleotide polymorphisms]] (SNPs) and differentiate among strains, including bacterial species previously considered to be highly homogeneous from a genetic point of view
- [[Genotypic typing methods#Multi-Locus Sequence Typing (MLST)|Multi-Locus Sequence Typing]] adapted to core or whole-genome genes (cgMLST/wgMLST) expands the loci comparison scope to hundreds or thousands, enhancing strain differentiation and source attribution effort

# Result
- offer robust, high-level [[Phylogeny|phylogenetic]] resolution [@efsapanelonbiologicalhazardsefsabiohazpanel.etal2019]


# WGS For [[Foodborne Diseases|foodborne disease outbreak]] investigation
[@efsapanelonbiologicalhazardsefsabiohazpanel.etal2019]
- main assumption: low genetic differences imply recent transmission or a common source
- WGS can help identifying the geographical origin and/or animal reservoir of an outbreak strain by analyzing epidemiological data associated with cases in phylogenetically related subclusters
	- [@mikhail.etal2018]: [[Escherichia coli|STEC]]
	  case-case study & epidemiological data & phylogenetic analysis to identify <u>ovine</u> exposures as a potential source of contamination. Mapping sheep and lamb movements helped establish epidemiological links between cases reporting ovine exposures, suggesting a geographical region associated with the outbreak strain
	- [@jenkins.etal2019]: review on the impact of WGS on the investigation of [[Escherichia coli|STEC]] outbreaks in England from 2013 to 2017
	- Siira et al., 2019: 

## Standardization
A working Group ISO TC34/SC9/WG25 at ISO developed an international standard on WGS typing and genomic characterization of foodborne bacteria [@mughini-gras.etal2019]

# Footer
## Related Notes
[@jackson.etal2016]
[@zhang.etal2019]

## Sources
-  [@mughini-gras.etal2019]
	- *Sheppard, S. K., Jolley, K. A., and Maiden, M. C. (2012). A gene-by-gene approach to bacterial population genomics: whole genome MLST of campylobacter. Genes 3, 261â€“277. doi: 10.3390/genes3020261*
	- *Moura, A., Criscuolo, A., Pouseele, H., Maury, M. M., Leclercq, A., Tarr, C., et al. (2016). Whole genome-based population biology and epidemiological surveillance of Listeria monocytogenes. Nat. Microbiol. 2:16185. doi: 10.1038/ nmicrobiol.2016.185*